<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduWidgetLOMFrV1</class>
 <widget class="QWidget" name="AbulEduWidgetLOMFrV1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
 border: 1px solid gray;
 border-radius: 5px;
 margin-top: 2ex;
}
 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center;
     padding: 0 3px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gbLOMFRRequises">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Indexation LOMFR</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QLabel" name="lblLOMFR1_3">
        <property name="text">
         <string>1.1 Identifiant</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblLOMFR1_1_1">
        <property name="text">
         <string>1.1.1 Catalogue </string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="leLOMFR1_1_1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Valeur URI (Uniform Resource Identifier) contrainte</string>
        </property>
        <property name="text">
         <string>URI</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="lblLOMFR1_1_2">
        <property name="text">
         <string>1.1.2 Entrée </string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="leLOMFR1_1_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Entrez une adresse http:// valide</string>
        </property>
        <property name="text">
         <string>Fournie par AbulÉduMédiathèque</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblLOMFR1_2">
        <property name="text">
         <string>1.2 Titre </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QLineEdit" name="leLOMFR1_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Saisissez le titre que portera le module. Il sera suffixé .abe</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QComboBox" name="cbLOMFR1_2Lang"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblLogoLOMFR">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="AbulEduWidgetLOM.qrc">:/abuleduwidgetlom/icones/lomfr.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblCommentaireLOMFR">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ce Logiciel respecte la norme LOMFR (Learning Object Metadata) permettant de décrire les ressources produites facilitant ainsi leur indexation.&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Les documents produits sont ainsi plus facile à retrouver.&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Vous devez renseigner manuellement des informations précises quand à la ressource que vous produisez, condition indispensable à une indexation correcte ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frAuthenticate">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblAuthenticate">
        <property name="text">
         <string>Vous devez être authentifié pour modifier ou compléter ces informations</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="hsAutheticate">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnAuthenticate">
        <property name="text">
         <string>S'authentifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>860</width>
       <height>338</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="AbulEduWidgetLOM.qrc"/>
 </resources>
 <connections/>
</ui>
